src/helpers.ts

```diff
@@ -142,7 +142,10 @@ export const request = async <T = Response>(
   options?: RequestInit,
   retries = requestSettings.retries
 ): Promise<T> => {
-  const { userId, integrator } = ConfigService.getInstance().getConfig()
+  const currentVersion = version
```

Comment:
I guess we can use `version` without reassignment.

Reply:
Yeah, I'll remove this. This was a placeholder variable